test/test_quax.py

```diff
@@ -1,8 +1,6 @@
 import unittest
+import quax
 
-from quax.quax import is_whole_sentence, is_misparsed, has_illegal_chars, has_blacklist_words, \
```

Comment:
@LydiaKoerber ... kleine Besserwisserei: Wenn du entwickelst, tüftelsts & ausprobierst, ist es vollkommen in Ordnungen Funktionen und Klassen direkt zu importieren, z.B. `from paket import myfunc`. In C nennt man das bspw. namespace import (So was gibt es in jeder P.Sprache). Aber in Production-code muss das weg-refactored werden, also einfach `import paket .... x = paket.myfunc(...)`. Warum? Stelle dir vor, dass du irgendwann mal irgendwer eine Funktion `andereres_paket.myfunc`  in den Code einbauen will => Schlimmer Bug kann die Folge sein (Ist alles schon passiert ...)